### ISSUE TITLE
scroll not working using key up/down

### ISSUE THREAD

**Issue by max-SS** [hakib/MassAutocomplete#57]
Hello I wonder if the option of using the **arrow keys up/down** in case there **scroll**, are supported to move. Why me not working me...

**Comment by hakib** [comment]
Hey @max-SS, not sure I never had very long lists :)

I suspect that scrolling a list might require setting the focus to the list item (which might pose a problem). Can you set up a plunkr/jsbin to demonstrate the issue?

**Comment by max-SS** [comment]
yes. I share here a fork of an example, I added the displacement

http://plnkr.co/edit/ZKKtYaeW0TpFt7Tij9Fm?p=preview

**Comment by hakib** [comment]
I see the problem. As I suspected it wont be possible because scrolling require settings focus to the selected item -  this conflicts with how the directive work. 

In order for the directive to be "global" we trigger attach on focus and detach on blur.Once we move the focus to the selected element a blur event will be triggered on the input element and a detach event will follow.

**Comment by gowrisankarreddy** [comment]
I am also facing same issues could u please help me to fix

**Comment by hakib** [comment]
Hey, see the explination above about the scrolling "challenge".

I think that if you have very long lists you should look into ways of making it smaller rather then tackle the scrolling issue. For example you can require more letters before making a suggestion, sort results by relevance and limit results etc.
